src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig

```diff
@@ -85,9 +85,6 @@
         {%- if required -%}
             {%- set label_attr = label_attr|merge({class: (label_attr.class|default('') ~ ' required')|trim}) -%}
         {%- endif -%}
-        {%- if parent_label_class is defined -%}
-            {%- set label_attr = label_attr|merge({class: (label_attr.class|default('') ~ ' ' ~ parent_label_class)|trim}) -%}
```

Comment:
The same logic exists in the Foundation template: https://github.com/symfony/symfony/blob/b560ec4a1d11f7b7e78a4bb9ed75b84397222a26/src/Symfony/Bridge/Twig/Resources/views/Form/foundation_5_layout.html.twig#L256-L258

Should we remove it as well?

Reply:
done